compiler/transforms/snax_copy_to_dma.py

```diff
@@ -28,6 +28,42 @@
 from compiler.ir.tsl import TiledStridedLayout
 
 
+def get_total_size_op(source: SSAValue):
```

Comment:
This is code from `MatchSimpleCopy` factored out as we also need it if we emit a 1D transfer in `TransformCopy`